channels/pjsip/dialplan_functions_doc.xml

```diff
@@ -374,6 +374,30 @@
 			<para>Parse the contents of the provided variable as a URI and return a specified part of the URI.</para>
 		</description>
 	</function>
+	<function name="TRANSFERHANDLING" language="en_US">
+		<since>
+			<version>23.0.0</version>
+		</since>
+		<synopsis>
+			Set the transferhandling behavior for a SIP call.
+		</synopsis>
+		<syntax>
+			<parameter name="mode" required="false">
+				<para>The transferhandling behavior to use. Default is <literal>default</literal>.</para>
```

Comment:
How transfers are handled for a PJSIP channel.

I think the default should actually be named "core" to indicate that the Asterisk core will handle them automatically.